HomeWork/src/main/java/multithreading/Main.java

```diff
@@ -0,0 +1,17 @@
+package multithreading;
+
+import java.util.Arrays;
+import java.util.HashMap;
+
+public class Main {
+    public static void main(String[] args) throws InterruptedException {
+
+        ValueCalculator valueCalculator = new ValueCalculator();
+        Thread thread1 = new Thread(valueCalculator, " Thread 1");
+        Thread thread2 = new Thread(valueCalculator, " Thread 2");
+
+        thread1.start();
+        thread2.start();
```

Comment:
It seems the main thread doesn't wait until these two will complete their jobs. Think about using `join` to solve this